ui/ducks/send/send.js

```diff
@@ -2486,6 +2559,11 @@ export function getRecipientUserInput(state) {
   return state[name].recipientInput;
 }
 
+export function getRecipientWarningAcknowledgement(state) {
+  return getCurrentDraftTransaction(state).recipient
+    .recipientWarningAcknowledged;
```

Comment:
Should probably use optional chaining with a sensible default (false?) because the draftTransaction *could be an empty object* if no draft transaction is currently found. This is an edge case but its better to guard against it.

Reply:
done in ca614adcb